data/templates/postfix/postsrsd

```diff
@@ -0,0 +1,43 @@
+# Default settings for postsrsd
+
+# Local domain name.
+# Addresses are rewritten to originate from this domain. The default value
+# is taken from `postconf -h mydomain` and probably okay.
+#
+SRS_DOMAIN={{ main_domain }}
```

Comment:
could you test again this syntax for the domain : because for me, I get an error : domain1.tld: NOT FOUND

Reply:
I just retested this tonight and for me it works ... where and when exactly do you see this error popping up ?